adminSiteServer/mail.ts

```diff
@@ -18,6 +18,7 @@ const transporter = nodemailer.createTransport({
     },
 })
 
+// get rid of this?
 export async function sendMail(
     options: nodemailer.SendMailOptions
 ): Promise<any> {
```

Comment:
It was only used for sending the invite. We can potentially remove the `nodemailer` dependency and remove the `EMAIL` global variables. (Although I'm not sure if Wordpress depends on them.)

Reply:
Do it!